### PR TITLE
Firefox / Thunderbird

### PR DESCRIPTION
**Pull request by APMichael**
Merged Firefox-based and Thunderbird-based entries. (Like most of the Chrome-based entries.)
Added support for Waterfox.

### CODE REVIEW

File: Winapp2.ini [hunk 1]
```diff
@@ -937,125 +875,131 @@ RegKey1=HKLM\Software\FullCircle\TalkBack
...
+FileKey13=%LocalAppData%\Moonchild Productions\Pale Moon\Profiles\*|ShutdownDuration.*
+FileKey14=%LocalAppData%\Mozilla\Firefox\Profiles\*|ShutdownDuration.*
+FileKey15=%LocalAppData%\Mozilla\SeaMonkey\Profiles\*|ShutdownDuration.*
+FileKey16=%LocalAppData%\Waterfox\Profiles\*|ShutdownDuration.*
 
 [Temps *]
 LangSecRef=3026
 SpecialDetect=DET_MOZILLA
 Default=False
-FileKey1=%LocalAppData%\VirtualStore\Program Files*\Mozilla*|*.tmp|RECURSE
```
Comment (MoscaDotTo) [inline Winapp2.ini]: Why was this line removed? Does the system not create this directory when the user account control is enabled?

Comment (APMichael) [reply Winapp2.ini]: I removed the few VirtualStore lines because I never had any files or folders under VirtualStore (on all my Windows 7, 8.1 and 10 systems). My UAC is always set to the highest level.

IMHO most of the VirtualStore lines aren't necessary. Only very old applications (coded for Windows XP and older) are redirected to VirtualStore.

Comment (ROCKNROLLKID) [reply Winapp2.ini]: So this isn't valid for version 52esr for XP users?

Comment (APMichael) [reply Winapp2.ini]: No, because Windows XP doesn't have the VirtualStore feature at all. This was introduced with Windows Vista for better compatibility with the new UAC.

Firefox was fully compatible with Windows Vista and higher at all time. Therefore, VirtualStore isn't used.

Comment (MoscaDotTo) [reply Winapp2.ini]: Thanks for the clarification :)

**Comment by ROCKNROLLKID** [comment]
I am thinking about renaming the sections now that we cover more then just "Chrome/Chromium" or "Firefox". Maybe renaming them to "Mozilla based browsers" and "Chrome/Chromium based Browsers"?
